py/torch_tensorrt/dynamo/conversion/impl/normalization/ops.py

```diff
@@ -48,89 +49,160 @@ def batch_norm(
     # Save the original output shape for later use
     output_shape = input.shape
 
-    # We name the weight here according to the state_dict name
-    weight = (
-        get_trt_tensor(ctx, 1.0, f"{name}_weight")
-        if weight is None
-        else get_trt_tensor(ctx, weight, f"{name}_weight")
-    )
-    bias = (
-        get_trt_tensor(ctx, 0.0, f"{name}_bias")
-        if bias is None
-        else get_trt_tensor(ctx, bias, f"{name}_bias")
-    )
-    running_mean = (
-        get_trt_tensor(ctx, 0.0, f"{name}_running_mean")
-        if running_mean is None
-        else get_trt_tensor(ctx, running_mean, f"{name}_running_mean")
-    )
-    running_var = (
-        get_trt_tensor(ctx, 1.0, f"{name}_running_var")
-        if running_var is None
-        else get_trt_tensor(ctx, running_var, f"{name}_running_var")
-    )
+    if all(
+        [
+            isinstance(weight, torch.Tensor),
+            isinstance(bias, torch.Tensor),
+            isinstance(running_mean, torch.Tensor),
+            isinstance(running_var, torch.Tensor),
+        ]
+    ):
+        if weight is None:
+            weight = 1.0
+
+        if bias is None:
+            bias = 0.0
+
+        if running_mean is None:
```

Comment:
add a docstring in this function explaining the batchnorm folding and why it is the default behavior for performance reasons for future reference.